code/modules/clothing/masks/gasmask.dm

```diff
@@ -100,13 +100,13 @@
 	actions_types = list(/datum/action/item_action/toggle_mask, /datum/action/item_action/change_appearance_mask, /datum/action/item_action/toggle_voicechanger)
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
 	var/list/clothing_choices = list()
-
+//someone who isn't me should really refactor this so it isnt instantiating all the masks in the game on startup
```

Comment:
Well then let me tell you how to do that!
There's an undocumented "feature" with `initial()` that lets you get the compile-time variables of a type without instantiating it, as long as none of said variables are lists.
Do the following:
1. Make a variable and cast it as the type you want to get the variables from (or a parent of it that has all the variables you need to get).
2. Assign to that variable the path of the *exact* type whose variables you want to get.
3. Use `initial()` on that variable as you normally would.

So replace the contents of the `for` loop below with something like `var/obj/item/clothing/mask/holder = Type` and wrap the variable reads in `change()` in `initial()`